homeassistant/components/netatmo/select.py

```diff
@@ -42,17 +42,28 @@ async def async_setup_entry(
     if not home_data or home_data.raw_data == {}:
         raise PlatformNotReady
 
-    if HOMEDATA_DATA_CLASS_NAME not in data_handler.data:
-        raise PlatformNotReady
+    for home_id in get_all_home_ids(home_data):
+        hass.data[DOMAIN][DATA_SCHEDULES].update(
```

Comment:
If there's only one key that needs to be updated we can just assign that key in the existing dict without creating a new dict and using dict.update.

Alternatively use a nested dict comprehension to create the whole new dict first and then update the existing dict with that.

Reply:
Maybe create a helper function, that updates the schedules, that we can use in both platforms?

Reply:
Of course both suggestions make sense. Thx